Added README.md (#19358)
github/docs: assets/images/README.md

> # Images🖼️

Reviewer: ```suggestion
# Images
```
While GitHub is emoji friendly, suggesting we remove it in this readme 😃

The `/assets/images` directory holds all the site's images.

> See the [contributing docs](/CONTRIBUTING.md) for general information about working with the docs.

Reviewer: ```suggestion
See [imaging and versioning](https://github.com/github/docs/blob/main/contributing/images-and-versioning.md) from the contributing docs for more information.
```
Awesome job linking users to the contributing.md ⚡
It would be great to take it just that extra step further and link them to the actual imaging md, so I've adjusted the link.
